example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -112,4 +166,202 @@ describe('the sf-starter lambda function', () => {
       expect(executions.length).toBe(messagesConsumed);
     });
   });
+
+  xdescribe('when provided a queue with a maximum number of executions', () => {
```

Comment:
Given the complications with integration testing, I'm going to disable these tests for now but leave them in. I'll pick them back up as part of 812